optuna/visualization/matplotlib/_edf.py

```diff
@@ -37,6 +37,12 @@ def plot_edf(
         Please refer to :func:`optuna.visualization.plot_edf` for an example,
         where this function can be replaced with it.
 
+    .. note::
+
+        Please refer `matplotlib.pyplot.legend
```

Comment:
Thank you for your update!

A nit, but I usually see `refer to the document` instead of `refer the document`.
```suggestion
        Please refer to `matplotlib.pyplot.legend
```

Reply:
oops. Thank you for correcting it!